src/main/pg/rcdevice.h

```diff
@@ -26,6 +26,10 @@
 typedef struct rcdeviceConfig_s {
     uint8_t initDeviceAttempts;
     timeMs_t initDeviceAttemptInterval;
+
+    // sometimes FC can't get featureInfo from devie(still no idea), so user can set it manaually.
```

Comment:
(device)

Reply:
ah, you mean is make the separate unittest and the logic code with different PR? But if I do it as you said, the 'make test' will failed in this PR..

Reply:
(Assume this is comment to ' PR if refactoring is in separate commit' )

Don't worry about it now, just consider it when opening new PR.

The idea was to commit refactoring changes in separate commit (protocolVer -> protocolVersion). There can be multiple commits in one PR and they can be displayed separately.